src/app/containers/FrontPageMain/index.jsx

```diff
@@ -44,22 +44,51 @@ const FrontPageMain = ({ frontPageData }) => {
         openGraphType="website"
       />
       <LinkedData type="WebPage" seoTitle={seoTitle} />
-      <main role="main">
-        <VisuallyHiddenText id="content" tabIndex="-1" as="h1">
-          {offScreenText}
-        </VisuallyHiddenText>
-        <Grid>
-          <GridItemConstrainedLargeWithTopMargin>
```

Comment:
Removing this makes the Grid to not have paddings on the sides. Do we need to also add [this](https://github.com/bbc/simorgh/blob/latest/src/app/lib/layoutGrid/index.js#L60) to the `src/app/components/Grid/index.jsx`?

Reply:
The usage of the extensions to `Grid`, e.g. `GridMaxWidthGroup4` appears to lose all of the prop logic that is built into `Grid`. I'm investigating why this is the case.

Reply:
Way to resolve this was to use the `forwardedAs` prop instead of the `as` prop. https://github.com/styled-components/styled-components/issues/1981#issuecomment-548860710
https://www.styled-components.com/docs/api#forwardedas-prop